package.json

```diff
@@ -30,12 +30,14 @@
     "start": "tsdx watch",
     "build": "tsdx build",
     "test": "tsdx test",
+    "testw": "tsdx test --watch -t Validate",
     "lint": "tsdx lint src test --ignore-pattern node_modules",
     "lint-fix": "yarn lint --fix",
     "ui": "yarn upgradeInteractive; cd example; yarn ui; yarn upgrade",
     "predeploy": "cd example; yarn; yarn build",
     "deploy": "gh-pages -d example/dist",
     "prepublish": "yarn build",
+    "prepare": "yarn build",
```

Comment:
Can you explain the effect this has?

Reply:
I had massive trouble trying to develop this locally. My main project is a CRA based app. For some reason `npm link` causes many problems with hooks and there is no clear fix for this. Then I wanted to just use my forked version off of github. But thats another problem of npm/yarn that they for some reason don't correctly build when installing. But everything that I read indicates that `prepare` is what _actually_ should be called when a package requires a compilation step.

Generally; would love to know how you develop this library locally while using it in another project. I resorted to just do TDD and hope it works ;)

Reply:
Sorry for the troubles! I just use the /example app and TDD. =) My own projects that depend on this library, I just rely on the published npm version.

If you `yarn start` in both the top level and in another window `yarn start` in the example folder, you should get the edit/reload experience. I'll clarify that in the readme.

The only time I have ever needed to build is when I explicitly call it or publish to npm (which uses prepublish).

I think for now, should be safe to remove this line if that is ok with you! =)

Reply:
I should add, I like forcing myself to use TDD with this project because it makes me feel more confident that upgrades to dependencies (primarily MUI/RFF) don't screw anything up. I only use the example app to get a visual test that things are looking ok.

Reply:
no worries, the problem has nothing to do with this lib but with CreateReactApp and React itself. But yes, TDD turned out to be good enough :)

I removed the prepare statement